index.php

```diff
@@ -149,6 +149,7 @@
                               ]),
         'languages_dropdown'  => Dropdown::showLanguages('language', [
             'display'             => false,
+            'rand'                => '1',
```

Comment:
Would be better to use `{% set rand = random() %}` in the template.
```suggestion
```

Reply:
An therefore pass it to all php dropdown methods calls?

Reply:
Probably, yes.